pyproject.toml

```diff
@@ -67,3 +67,51 @@ select = [
 "**/{tests}/*" = ["B007"]
 "__init__.py" = ["F401"]
 "docs/conf.py" = ["F401"]
+
+[tool.pixi.project]
+name = "jaxsim"
+channels = ["conda-forge"]
+platforms = ["linux-64", "linux-aarch64", "osx-64", "osx-arm64"]
+
+[tool.pixi.dependencies]
+coloredlogs = "*"
+jax = "*"
+jaxlib = "*"
+jaxlie = "*"
+jax-dataclasses = "*"
+pptree = "*"
+rod = "*"
+typing_extensions = "*"
+lxml = "*"
+mediapy = "*"
+mujoco = "*"
+libgz-sim8 = "*"
+
+[tool.pixi.feature.test.tasks]
+test = "pytest"
+
+[tool.pixi.feature.test.dependencies]
+black = "24.*"
+isort = "*"
+pre-commit = "*"
+idyntree = "*"
+pytest = "*"
+pytest-icdiff = "*"
+robot_descriptions = "*"
+
+[tool.pixi.feature.gpu]
+
+platforms = ["linux-64"]
+system-requirements = {cuda = "12.1"}
+# cuda-nvcc and cuda-cupti are just a workaround for
+# https://github.com/conda-forge/jaxlib-feedstock/pull/241
+dependencies = {cuda-version = "12.*", cuda-nvcc  = "*", cuda-cupti = "*", jaxlib = "* *cuda*"}
```

Comment:
If we pin a minimum `jaxlib` version in `tool.pixi.dependency`, should we also copy it here?

Reply:
I guess so.

Reply:
Ok, I was wondering whether we have to duplicate it if a range is specified.